EIPS/eip-2015.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 2015
+title: Wallet Update Chain JSON-RPC Method (`wallet_updateChain`)
+author: Pedro Gomes (@pedrouid)
+discussions-to: https://ethereum-magicians.org/t/add-wallet-methods-to-improve-dapp-to-wallet-interaction/1848/31
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-05-12
+requires: 155
```

Comment:
Oh, would also 1474 here would make sense (https://eips.ethereum.org/EIPS/eip-1474) since it defines the RPC basics?